components/table/table-header.js

```diff
@@ -6,7 +6,17 @@ import { SelectionHeader } from '../selection/selection-header.js';
 /**
  * A header for table components containing a selection summary and selection actions.
  */
-class TableHeader extends SelectionHeader {
```

Comment:
Did you intend to make these changes with this PR? Don't seem related/needed.

Reply:
Ah! Yes mostly, though the `export` isn't necessary.

This doc needed updating for `TableHeader` specifically, so I've overridden the underlying `noSelection` prop to change its description.